# HG changeset patch
# User jpclizbe
# Date 1369957957 18000
# Node ID f80dd03fc3dfaface260284b6c053cbe7a7a1148
# Parent  ad3d160cfa4d60a55805465aebe56316fa60682f
Replaced occurrances of the deprecated operator 'or' with '||'

As reported by Damien Doligez in https://bitbucket.org/skskeyserver/sks-keyserver/issue/2/use-of-deprecated-operator-or

diff --git a/mArray.ml b/mArray.ml
--- a/mArray.ml
+++ b/mArray.ml
@@ -57,7 +57,7 @@
   Array.fold_left ~f:(fun a b -> a && (test b)) ~init:true array
 
 let exists ~f:test array =
-  Array.fold_left ~f:(fun a b -> a or (test b)) ~init:false array
+  Array.fold_left ~f:(fun a b -> a || (test b)) ~init:false array
 
 let mem el array =
   let length = Array.length array in
diff --git a/prime.ml b/prime.ml
--- a/prime.ml
+++ b/prime.ml
@@ -87,7 +87,7 @@
   let test () =
     let a = randrange rfunc two (n -! one) in
     let y = Number.powmod a r n in
-    if y =! one or y =! neg_one then Prime
+    if y =! one || y =! neg_one then Prime
     else
       let rec loop y j =
         if y =! neg_one then Prime
